app/assets/javascripts/admin/addon/components/site-settings/category-list.gjs

```diff
@@ -21,14 +21,22 @@ export default class CategoryList extends Component {
     return this.args.value.split("|").filter(Boolean);
   }
 
-  async updateSelectedCategories() {
-    await this.pendingCategoriesRequest;
-    this.selectedCategories = await Category.asyncFindByIds(this.categoryIds);
+  async updateSelectedCategories(previousRequest) {
+    let categories = await Category.asyncFindByIds(this.categoryIds);
```

Comment:
```suggestion
    const categories = await Category.asyncFindByIds(this.categoryIds);
```

Since we're not modifying `categories`, a constant should work fine here.